apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js

```diff
@@ -31,6 +31,18 @@ function SubmissionForm() {
     ['primary', 'close'],
   );
 
+  const getInvolveLevel = (involveString) => {
+    if (involveString.includes("and also believe")) {
```

Comment:
Maybe it'll be easier for other developers to maintain if this mapping is defined closer where the strings are defined.

Reply:
Agreed. Rutuja will be implementing the long term fix here including the UI issue noted above.